fs/fs.go

```diff
@@ -373,7 +374,7 @@ type filesystem struct {
 }
 
 func (fs *filesystem) MountLocal(ctx context.Context, mountpoint string, labels map[string]string, mounts []mount.Mount) error {
-	imageRef, ok := labels[source.TargetRefLabel]
+	imageRef, ok := labels[ctdsnapshotters.TargetRefLabel]
```

Comment:
this will be a behavior change right? since `source.TargetRefLabel` and `ctdsnapshotters.TargetRefLabel` are different strings?